src/coreclr/pal/src/misc/perfjitdump.cpp

```diff
@@ -2,15 +2,15 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // ===========================================================================
 
-#if defined(__linux__)
-#define JITDUMP_SUPPORTED
-#endif
-
 #include "pal/palinternal.h"
 #include "pal/dbgmsg.h"
 
 #include <cstddef>
 
+#if defined(__linux__) || (defined(__APPLE__) && defined(TARGET_OS_OSX) && TARGET_OS_OSX)
```

Comment:
```suggestion
#ifdef FEATURE_PERFMAP
```
Would this work?

Reply:
I don't seem to have the `FEATURE_*` definitions available here. The jitdump code is part of the PAL, and from what I can see the cmake FEATURE_* definitions don't end up applying there?